.gitignore

```diff
@@ -7,4 +7,5 @@
 /pkg/
 /spec/reports/
 /tmp/
-*.gem
\ No newline at end of file
+*.gem
+/.idea/
```

Comment:
lol that's good one. ha ha. plus, it saves you from having to add it to every new project's gitignore